packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.test.js

```diff
@@ -1,14 +1,144 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
-import DateTimeView from './DateTimeView.component';
+import DateTimeView, { euclideanModulo } from './DateTimeView.component';
 
 describe('DateTimeView', () => {
-	it('should render a DateTimeView', () => {
-		// when
-		const wrapper = shallow(<DateTimeView />);
+	it('should render', () => {
+		const wrapper = shallow(<DateTimeView
+			selectedMonthIndex={5}
+			selectedYear={2006}
+		/>);
 
-		// then
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+
+	it('should callback when title is clicked', () => {
+		const onClickTitle = jest.fn();
+		const wrapper = shallow(<DateTimeView
+			selectedMonthIndex={5}
+			selectedYear={2006}
+			onClickTitle={onClickTitle}
+		/>);
+
+		const titleAction = wrapper
+			.find('ViewLayout')
+			.shallow()
+			.find('HeaderTitle')
+			.first()
+			.shallow()
+			.find('button');
+
+		titleAction.simulate('click');
+		expect(onClickTitle).toHaveBeenCalledTimes(1);
+	});
+
+	describe('should callback when month switched with the new month index and year', () => {
```

Comment:
`describe()` should not describe a test case, but a test group.

`describe` month switch management
`it` should trigger callback with previous month
`it` should trigger callback with previous month, changing the year
`it` should trigger callback with next month
`it` should trigger callback with next month, changing the year

So in your test log you'll have 
`DateTimeView month switch management should trigger callback with previous month`
And if you read your test `it should trigger callback with previous month` (and not `it in simple previous case`)

Reply:
✔